go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/matchsystems/werr
 go 1.21.4
 
 require (
+	github.com/go-errors/errors v1.5.1
```

Comment:
Probably we should move benchmark to another module, because werr not require errorx or go-errors